Framework/CurveFitting/src/IMWDomainCreator.cpp

```diff
@@ -412,6 +413,11 @@ void IMWDomainCreator::addFunctionValuesToWS(
     const std::shared_ptr<API::FunctionValues> &resultValues) const {
   const size_t nData = resultValues->size();
   resultValues->zeroCalculated();
+  // Confidence band of 1 sigma
+  double sigma = 1;
```

Comment:
Maybe we could provide some comments/references on the error calculations r.e. where the theory here has come from so in future this confusion doesn't crop up?